lib/language.js

```diff
@@ -54,10 +54,14 @@ exports.errors = {
         length: 'must be {{limit}} bytes'
     },
     date: {
-        base: 'must be a number of milliseconds or valid date string',
+        base: 'must be a valid date string or timestamp',
```

Comment:
I'll agree to disagree here. Since you're the maintainer, ultimately it's your decision. If this is that important to you I'll update it and rebase so you'll have a clean merge. What do you want to do?